.github/actions/entrypoint.sh

```diff
@@ -2,7 +2,12 @@
 
 # Exit when any command fails
 set -e
-
+set -x
 cd "$GITHUB_WORKSPACE"
-"$GITHUB_WORKSPACE/ci/master.bash" QUICK
-"$GITHUB_WORKSPACE/ci/master.bash" STATIC
+if [ "$GITHUB_WORKFLOW" != "Autodocs" ]
```

Comment:
Nitpick: If you're overriding the entrypoint in the autodocs workflow, do we also need this conditional to swap the action?

Reply:
Yes.  The entrypoint, although documented on actions, doesn't work(?)  so I have to fallback to this.  Regardless, this setup functions in my repo, and given the complexity and time invested in getting this far, I really don't want to change it.